easybuild/easyconfigs/r/R/R-3.3.0-foss-2016a.eb

```diff
@@ -0,0 +1,455 @@
+    ('BatchJobs', '1.6', ext_options),
+    ('sandwich', '2.3-4', ext_options),
+    ('sfsmisc', '1.1-0', ext_options),
+    ('spatial', '7.3-11', ext_options),
+    ('VGAM', '1.0-1', ext_options),
+    ('waveslim', '1.7.5', ext_options),
+    ('xtable', '1.8-2', ext_options),
+    ('profileModel', '0.5-9', ext_options),
+    ('brglm', '0.5-9', ext_options),
+    ('deSolve', '1.13', ext_options),
+    ('tseriesChaos', '0.1-13', ext_options),
+    ('tseries', '0.10-35', ext_options),
+    ('fastICA', '1.2-0', ext_options),
+    ('R.methodsS3', '1.7.1', ext_options),
+    ('R.oo', '1.20.0', ext_options),
+    ('cgdsr', '1.2.5', ext_options),
+    ('R.utils', '2.3.0', ext_options),
+    ('R.matlab', '3.5.1', ext_options),
+    ('gbm', '2.1.1', ext_options),
+    ('dichromat', '2.0-0', ext_options),
+    ('Formula', '1.2-1', ext_options),
+    ('acepack', '1.3-3.3', ext_options),
+    ('reshape2', '1.4.1', ext_options),
+    ('gtable', '0.2.0', ext_options),
+    ('munsell', '0.4.3', ext_options),
+    ('labeling', '0.3', ext_options),
+    ('scales', '0.4.0', ext_options),
+    ('proto', '0.3-10', ext_options),
+    ('ggplot2', '2.1.0', ext_options),
+    ('gridExtra', '2.2.1', ext_options),
+    ('chron', '2.3-47', ext_options),
+    ('data.table', '1.9.6', ext_options),
+    ('Hmisc', '3.17-4', ext_options),
+    ('fastcluster', '1.1.20', ext_options),
+    ('registry', '0.3', ext_options),
+    ('pkgmaker', '0.22', ext_options),
+    ('rngtools', '1.2.4', ext_options),
+    ('doParallel', '1.0.10', ext_options),
+    ('gridBase', '0.4-7', ext_options),
+    ('NMF', '0.20.6', ext_options),
+    ('irlba', '2.0.0', ext_options),
+    ('igraph', '1.0.1', ext_options),
+    ('GeneNet', '1.2.13', ext_options),
+    ('ape', '3.4', ext_options),
+    ('htmltools', '0.3.5', ext_options),
+    ('RJSONIO', '1.3-0', ext_options),
+    ('caTools', '1.17.1', ext_options),
+    ('gplots', '3.0.1', ext_options),
+    ('ROCR', '1.0-7', ext_options),
+    ('httpuv', '1.3.3', ext_options),
+    ('R6', '2.1.2', ext_options),
+    ('jsonlite', '0.9.20', ext_options),
+    ('rjson', '0.2.15', ext_options),
+    ('shiny', '0.13.2', ext_options),
+    ('seqinr', '3.1-3', ext_options),
+    ('LearnBayes', '2.15', ext_options),
+    ('deldir', '0.1-12', ext_options),
+    ('gmodels', '2.16.2', ext_options),
+    ('spdep', '0.6-4', ext_options),
+    ('assertthat', '0.1', ext_options),
+    ('lazyeval', '0.1.10', ext_options),
+    ('dplyr', '0.4.3', ext_options),
+    ('vegan', '2.3-5', ext_options),
+    ('adegenet', '2.0.1', ext_options),
+    ('rncl', '0.6.0', ext_options),
+    ('XML', '3.98-1.4', ext_options),
+    ('memoise', '1.0.0', ext_options),
+    ('crayon', '1.3.1', ext_options),
+    ('praise', '1.0.0', ext_options),
+    ('testthat', '1.0.2', ext_options),
+    ('yaml', '2.1.13', ext_options),
+    ('knitr', '1.13', ext_options),
+    ('rmarkdown', '0.9.6', ext_options),
+    ('curl', '0.9.7', ext_options),
+    ('openssl', '0.9.3', ext_options),
```

Comment:
this probably mean we need an (OS) dep on OpenSSL?

cfr. https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/p/Python/Python-2.7.11-intel-2016a.eb#L26

Out of curiosity: did you include this as a dep for another extension?